P5/Source/Specs/localName.xml

```diff
@@ -27,13 +28,22 @@ $Id$
     <textNode/>
     <!--s:assert> some schematron rules here</s:assert-->
   </content>
-  <exemplum xml:lang="mul">
+  <constraintSpec scheme="schematron" validUntil="2021-07-09"
```

Comment:
Delete the entire `<constraintSpec>`. The deprecation Schematron check will (or at least should) be automatically generated at build time because of the `@validUntil` on the `<elementSpec>`.